src/main/java/edu/hw1/Task1.java

```diff
@@ -0,0 +1,26 @@
+package edu.hw1;
+
+@SuppressWarnings("MagicNumber")
+public final class Task1 {
+    static int secondsInMinute = 60;
```

Comment:
`BUG` В джаве константы принято наименовывать так:
```suggestion
    static int SECONDS_IN_MINUTE= 60;
```
Хотя на мой взгляд, тут можно просто переменной а не константой